Basic-Car-Maintenance.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,113 +0,0 @@
-{
```

Comment:
Why is this file deleted?

Reply:
looks like was done by mistake from last merge, should be good now 👍